apps/omg_watcher_info/test/support/factories/pending_block_factory.ex

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.Factory.PendingBlock do
+  @moduledoc """
+    Pending Block factory.
+
+    Generates a pending block that will need to be processed and inserted to the database.
+  """
+
+  defmacro __using__(_opts) do
```

Comment:
pls avoid using macros. There's no added value for a macro here. extract it into public functions that you can call.

Reply:
👍 Just copied from other factories, will update all.

Reply:
FYI, this happened to be [ex_machina's recommended way](https://github.com/thoughtbot/ex_machina#splitting-factories-into-separate-files) and I had no energy left to debate back then, but I agree there's no need for de facto inheritance

Reply:
Ahaha, so I revert all factories to be simple imports.

Reply:
ah okay, whatever ex_machina says it right!